web_responsive/static/src/less/form_view.less

```diff
@@ -52,6 +52,24 @@
     }
 }
 
+// Sticky Header & Footer in List View
+.o_view_manager_content {
+    >div {
+        >.table-responsive {
+            >.o_list_view {
+                thead {
+                    position: sticky;
```

Comment:
sounds this is not going to work for everybody https://caniuse.com/#search=sticky :)

Reply:
Usually CSS would ignore unknown rules and just leave the defaults, so that shouldn't be a problem in most cases.

Reply:
sure, maybe worth to add it to known issue / roadmap?

Reply:
Well, it's not *known* until you open a old browser and check it's broken.
Let's just expect it works, and if it breaks somebody, and that somebody cannot stop using IE, we'll fix it then. :relaxed: